website/content/en/docs/concepts/nodepools.md

```diff
@@ -229,6 +259,11 @@ Karpenter can be configured to create nodes in a particular zone. Note that the
 [Learn more about Availability Zone
 IDs.](https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html)
 
+**Static NodePool**
+
+Topology requirement field is the source of truth for topology decisions. Users who want to spread nodes across zones can do so explicitly by:
+- Creating multiple static NodePools, each pinned to a specific AZ.
```

Comment:
I'd still move this to the examples section because it's not really part of the definition of the label, if we want to include it at all. Also, topology doesn't only refer to zonal requirements. Generally speaking, and k8s label defines a topology. I'd avoid using the term topology when specifically referring to zonal requirements.